data/src/main/java/smile/data/parser/microarray/PCLParser.java

```diff
@@ -127,7 +129,7 @@ public AttributeDataset parse(File file) throws IOException, ParseException {
      * @throws java.io.IOException
      */
     public AttributeDataset parse(String name, File file) throws IOException, ParseException {
-        return parse(name, new FileInputStream(file));
+        return parse(file);
```

Comment:
Do we discard the name here on purpose?

Reply:
It was a change done by you to fix compilation errors I think. @jmarten

Reply:
I can add it back if it was by accident